components/bl-avatar-generator/src/helpers.js

```diff
@@ -0,0 +1,223 @@
+import { data as characterOptions } from './avatar-data';
+
+const RANDOM_OPTION = 'Random';
+
+const OptionsKeys = {
+  BACKGROUND        : 'background',
+  CIRCLE_COLOR      : 'circleColor',
+  SKIN              : 'skinColors',
+  TOP               : 'topTypes',
+  HAIR_COLOR        : 'hairColors',
+  HAT_COLOR         : 'hatColors',
+  BROWS             : 'browsTypes',
+  EYES              : 'eyesTypes',
+  MOUTH             : 'mouthTypes',
+  FACIAL_HAIR       : 'facialHairTypes',
+  FACIAL_HAIR_COLOR : 'hairColors',
+  ACCESSORY         : 'accessoryTypes',
+  CLOTHES           : 'clothesTypes',
+  FABRIC_COLOR      : 'hatColors',
+  GRAPHIC           : 'graphicTypes',
+};
+
+// the order is important for handleRandomOptions
```

Comment:
https://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order/38218582#38218582

A should know that you familiar with it, before approve this part of code.